cloudify_agent/resources/script/windows.ps1.template

```diff
@@ -54,14 +54,14 @@ function InstallAgent()
     {% if add_ssl_cert %}
     AddSSLCert
     {% endif %}
-    if (-Not (Test-Path "{{ conf.basedir }}\Scripts")) {
+    if (-Not (Test-Path "C:\Program Files\Cloudify {{ version }} Agents\Scripts")) {
```

Comment:
Just a general comment. Don't we want to assign "C:\Program Files\Cloudify {{ version }} Agents" to a variable and reuse it in all places?

Reply:
That seemed to be what conf.basedir was being partially used for; but it's also wrong so not useful for that. I'm checking to see if it crops up anywhere else.

Reply:
As for refactoring it to have a common variable that's right: Yes, we should probably do that but I'm thinking of holding off on that for the 5.1.1 release unless I find anything to indicate that basedir being wrong will cause flames and sadness.

Reply:
OK, I scrapped this bit of code, updated to a different way.